http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -132,10 +132,10 @@ class GceInterpreter[F[_]: Parallel: ContextShift](
               )
             )
             isFormatted <- persistentDisk.formattedBy match {
-              case Some(FormattedBy.Galaxy) =>
+              case Some(FormattedBy.Galaxy) | Some(FormattedBy.Custom) =>
                 F.raiseError[Boolean](
                   new RuntimeException(
-                    "Trying to use a Galaxy formatted disk for creating GCE runtime. This should never happen."
+                    "Trying to use an app formatted disk for creating GCE runtime. This should never happen."
```

Comment:
can we also include the actual app in the message

Reply:
I'll include the disk id. Should allow debugging with one extra step on devs part instead of a potential app lookup in the gceinterp, which seems wrong.